.storybook/webpack.config.js

```diff
@@ -33,7 +33,17 @@ module.exports = ({ config }) => {
           localIdentName: '[name]__[local]___[hash:base64:5]',
         },
       },
-      require.resolve('sass-loader'),
+      {
+        loader: require.resolve('sass-loader'),
+
+        options: {
```

Comment:
I'm wondering if we should put this definition in an export or somewhere so we don't have to manage it here and in the webpack config for storybook.

Reply:
That's a great idea. could be exported from gatsby-config.js, or create an external file. I'd rather make it an exported constant in gatsby-config.js for readability.